src/data/arithm.rs

```diff
@@ -457,6 +457,50 @@ impl Number {
             _ => Some(val),
         }
     }
+
+    /// Adds or removes negative sign to the number (negates negative or positive number, depending
+    /// on the method argument value)
+    ///
+    /// # Panics
+    ///
+    /// - if applied to unsigned integer layout
+    /// - if attempted to negate the minimum possible value for its layout (e.g. -128 as 1 byte)
+    #[inline]
+    pub fn applying_sign(mut self, sign: impl Into<bool>) -> Option<Number> {
+        let layout = self.layout();
+        match layout {
+            Layout::Integer(IntLayout { signed: true, .. }) => {
+                if !self.is_positive() ^ sign.into() {
```

Comment:
Nit: since we need to change docs anyway, may we pls raise this `if` on a level up to have two match cases, where the first one has `if ...`?

Reply:
thanks